frontend/javascripts/oxalis/view/settings/export_bounding_box_modal.js

```diff
@@ -31,12 +32,15 @@ const ExportBoundingBoxModal = ({ destroy, dataset, boundingBox }: Props) => {
   };
 
   const layerNames = dataset.dataSource.dataLayers.map(layer => {
-    const nameIfColor = layer.category === "color" ? layer.name : null;
+    const nameIfFromDataset = layer.category === "color" || !hasVolumeTracing ? layer.name : null;
     const nameIfVolume =
-      layer.category === "segmentation" && layer.fallbackLayerInfo && layer.fallbackLayerInfo.name
+      hasVolumeTracing &&
+      layer.category === "segmentation" &&
+      layer.fallbackLayerInfo &&
+      layer.fallbackLayerInfo.name
         ? layer.fallbackLayerInfo.name
         : null;
-    return nameIfColor || nameIfVolume;
+    return nameIfFromDataset || nameIfVolume;
```

Comment:
I find this code a bit hard to read. Using early-returns could avoid the duplicate check of the `category` for example:

```js
if (layer.category === "color" || !hasVolumeTracing) {
	return layer.name;
}

if (layer.fallbackLayerInfo != null) {
	return layer.fallbackLayerInfo.name;
}

// For a volume layer without fallback information, we cannot provide an export currently
return null;
```

However, it might be a matter of taste. Code usually reads better if `reader == writer` :laughing:

Reply:
I’ll include this feedback in the next version, which wil be a bit more complex :see_no_evil: , and which is already on the way in https://github.com/scalableminds/webknossos/pull/5246